packages/lesmis-server/src/utils/accessPolicy.ts

```diff
@@ -7,14 +7,15 @@ import { AccessPolicy } from '@tupaia/access-policy';
 import { PermissionsError } from '@tupaia/utils';
 import { LESMIS_COUNTRY_CODE, LESMIS_PERMISSION_GROUP } from '../constants';
```

Comment:
```suggestion
import { LESMIS_COUNTRY_CODE, LESMIS_BASE_PERMISSION_GROUP, LESMIS_ADMIN_PERMISSION_GROUP } from '../constants';
```
Rather than having `LESMIS Public` in a const and `LESMIS Admin` inline, let's pull them both into constants